src/JustWatch.elm

```diff
@@ -0,0 +1,216 @@
+module JustWatch exposing (..)
+
+import Json.Decode as Decode exposing (string, list)
+import Json.Decode.Pipeline exposing (decode, hardcoded, optional, required)
+import Http exposing (encodeUri)
+
+
+type MovieOffers
+    = Loading
+    | NoResults
+    | Results (List JustWatchOffer)
+
+
+type OfferType
+    = Buy
+    | Rent
+    | Streaming
+    | Ads
+    | Free
+    | Unknown
+
+
+movieSearchUrl : String -> String
+movieSearchUrl title =
+    "https://apis.justwatch.com/content/titles/en_US/popular?body=" ++ (Http.encodeUri ("{ \"query\": \" " ++ title ++ "\"}"))
```

Comment:
Really really minor, but this might be easier to read if the string concatenation came first and then the whole url was piped to `encodeUri`